sdk/storage/azure-storage-blob/v10_v12.md

```diff
@@ -0,0 +1,365 @@
+# Storage Blob Service SDK Migration Guide from 10.x/11.x to 12.x
+
+In this section, we list main changes you need to be aware of when converting your Storage Blob SDK library from Version 10/Version 11 to Version 12.
+
+## Prerequisites
+Java Development Kit (JDK) with version 8 or above
+
+## Update reactive framework from RxJava to Reactor and ReactorNetty
+For more info related to Reactor project, please refer [Reactor Project](https://projectreactor.io/)
+For more info of the motivation to V12, please refer [V12 Upgrade Storage](https://github.com/Azure/azure-storage-java/blob/master/V12%20Upgrade%20Story.md)
+
+## Converting Core Classes
+| Core v10/11 classes  | Equivalent v12 Class | NEW synchronous clients |
+|---|---|---|
+| ServiceURL  |  BlobServiceAsyncClient  | BlobServiceClient  |
+| ContainerURL |  ContainerAsyncClient | ContainerClient |
+| BlobURL    | BlobAsyncClient  |   BlobClient |
+| BlockBlobURL  |  BlockBlobAsyncClient  |  BlockBlobClient  |
+| AppendBlobURL  |  AppendBlobAsyncClient | AppendBlobClient  |
+| PageBlobURL |  PageBlobAsyncClient | PageBlobClient  |
+
+
+## Updated Maven dependency
+Dependency for Blob service:
+```xsd
+<dependency>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-storage-blob</artifactId>
+  <version>12.0.0-preview.2</version>
+</dependency>
+```
+
+## Generate SAS token
+You can build the SAS token through the client builder `endpoint`. Adds the sasToken from portal at the end of the endpoint. e.g. `https://${accountName}.blob.core.windows.net/?${sasToken}`
```

Comment:
It's unclear here what building the SAS token means
Does it mean creating a SAS token or building a new client with a SAS token?